Code/BasicFilters/json/IsolatedConnectedImageFilter.json

```diff
@@ -13,7 +13,7 @@
       "dim_vec" : 1,
       "itk_type" : "typename FilterType::IndexType",
       "briefdescriptionSet" : "",
-      "detaileddescriptionSet" : "Set a single seed point 1.\n\nThis seed will be isolated from Seed2 (if possible). All pixels connected to this seed will be replaced with ReplaceValue."
+      "detaileddescriptionSet" : "Set a single seed point 1.\n\nThis seed will be isolated from Seed2 (if possible). All pixels connected to this seed will be replaced with ReplaceValue.\n\nDeprecatedPlease use AddSeed1."
```

Comment:
"DeprecatedPlease". Should there be some separator between the words?

Reply:
This is odd that this method was deprecated. It looks like the ITK class needs a little clean up for standard setter/getter methods.